main/res/values/strings.xml

```diff
@@ -252,6 +253,8 @@
     <string name="info_stored_image">New image saved to:</string>
     <string name="info_storing_static_maps">Trying to store static maps</string>
     <string name="info_cache_saved">The cache has been stored locally</string>
+    <string name="info_calculator_state_saved">Current state of the calculator has been saved to waypoint.</string>
```

Comment:
IMHO there should be only a message in case of an error and no explicit save button.